### DISCUSSION TITLE
Convert SEG format and do CNVkit to analysis ?

### DISCUSSION THREAD

**Discussion by SEvisual** [etal/cnvkit#1021] [General]
I download TCGA CNV data(Masked Copy Number Segment, SEG format), and I wonder if I can do this to analysis CNV: 
1.using `cnvkit.py import-seg SEG_File` to convert it into CNVkit .cns files 
2.using `cnvkit.py call .cns` to analysis CNV or `cnvkit.py scatter` to plot

Is this right ? Hope you can tech me and  address my doubts.

Thank you for giving your soul!

**Comment by etal** [comment]
Yes, that looks right.
